django-spectre-css/templatetags/spectre_css.py

```diff
@@ -21,7 +21,7 @@ def is_select(field):
 
 @register.filter
 def is_multi_field(field: BoundField):
-    return "fields" in field.field.__dict__
+    return issubclass(type(field), MultiValueField)
```

Comment:
Would `isinstance(field, MultiValueField)` work and be cleaner?

Reply:
yes it was possible...